specification/support/resource-manager/Microsoft.Support/preview/2021-06-01-preview/supportResourceIdDetails.json

```diff
@@ -0,0 +1,291 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow."
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Support/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "This lists all the available Microsoft Support REST API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved list of operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationsListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "Get all operations": {
+            "$ref": "./examples/ListOperations.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Support/lookUpResourceId": {
+      "post": {
+        "tags": [
+          "SupportTickets"
+        ],
+        "summary": "This operation fetches ARM resource id of support resource type.",
+        "operationId": "LookUpResourceId_Post",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "lookUpResourceIdRequest",
+            "in": "body",
+            "description": "Look up resource id request body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LookUpResourceIdRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully obtained resource id of support resource type.",
+            "schema": {
+              "$ref": "#/definitions/LookUpResourceIdResponse"
+            }
+          },
+          "400": {
+            "description": "Request or parameter(s) is malformed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Access to support resource is forbidden.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "404": {
+            "description": "Support resource does not exist.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
```

Comment:
>default [](http://example.com/codeflow?start=11&length=7)

Explicitly enumerating failure status code has side effects on SDK generation and also misleads users into thinking that those are the only status code you can return. Both your endpoint and ARM itself can return other 4xx and 5xx status codes.

The best practice is to only model `200` and `default` for such an API.